tests/tablenavigation.js

```diff
@@ -1970,6 +2098,51 @@ describe( 'TableNavigation', () => {
 							[ '20', '21', '22' ]
 						] ) );
 					} );
+
+					describe( 'when shift key is pressed', () => {
+						beforeEach( () => {
+							upArrowDomEvtDataStub.shiftKey = true;
+							downArrowDomEvtDataStub.shiftKey = true;
+						} );
+
+						it( 'should expand selection to the beginning of the cell content', () => {
```

Comment:
Also for tests in wrap area - it would be nice to have also tests for expanded selection.

Reply:
Adressed in fedf508.